lib/core/presentation/widgets/poap/poap_claim_transfer_controller_widget/poap_claim_transfer_controller_widget.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:app/core/application/auth/auth_bloc.dart';
+import 'package:app/core/application/poap/poap_claim_transfer_subscription_bloc/poap_claim_transfer_subscription_bloc.dart';
+import 'package:app/core/domain/poap/poap_enums.dart';
+import 'package:app/core/presentation/widgets/poap/poap_claimed_poup/poap_claimed_popup.dart';
+import 'package:app/i18n/i18n.g.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class PoapClaimTransferControllerWidget extends StatelessWidget {
+  const PoapClaimTransferControllerWidget({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<AuthBloc, AuthState>(
+      builder: (context, state) => state.maybeWhen(
+        orElse: () => const SizedBox.shrink(),
+        authenticated: (_) => BlocProvider(
+          create: (context) => PoapClaimTransferSubscriptionBloc()
```

Comment:
Put BlocProvider above to create its instance only once, put it here may cause the app create multiple instance of it

Reply:
Bloc auto close its instance when not mounting any more, this only creates when ever user authenticated, and killed when user logged out, not seeing any issue with this ?